nussl/core/augmentation.py

```diff
@@ -0,0 +1,377 @@
+import numpy as np
+import numpy.random as random
+import librosa
+import os
+from .audio_signal import AudioSignal
+import tempfile
+from .augmentation_utils import *
```

Comment:
Import explicitly the functions you need. I think these utils should be moved into `augment.py` directly, actually.